src/dispatch.rs

```diff
@@ -68,15 +68,16 @@ impl AsyncDispatcher {
     }
 
     /// Waits for all the systems to finish
-    /// and executes thread local systems (if there
-    /// are any).
     pub fn wait(&mut self) {
         self.signal
             .take()
             .expect(ERR_NO_DISPATCH)
             .wait()
             .expect("The worker thread may have panicked");
+    }
```

Comment:
Because for example i just executed thread local systems just after async system dispatching and do not want to execute them again.